src/main/java/ru/bclib/client/gui/BlockSignEditScreen.java

```diff
@@ -5,12 +5,7 @@
 import com.mojang.blaze3d.platform.GlStateManager;
 import com.mojang.blaze3d.platform.Lighting;
 import com.mojang.blaze3d.systems.RenderSystem;
-import com.mojang.blaze3d.vertex.BufferBuilder;
-import com.mojang.blaze3d.vertex.BufferUploader;
-import com.mojang.blaze3d.vertex.DefaultVertexFormat;
-import com.mojang.blaze3d.vertex.PoseStack;
-import com.mojang.blaze3d.vertex.Tesselator;
-import com.mojang.blaze3d.vertex.VertexConsumer;
+import com.mojang.blaze3d.vertex.*;
```

Comment:
It's not good to use * imports, this will make logically hard to determine where are different methods called

Reply:
Agree. Will fix that, looks like Idea put those in.